[issue/7] - added motivation part (#29)
teamulster2/soTired: README.md

> # sotired

Reviewer: I'd keep the heading and adjust it to the repository name

# soTired

## Motivation

soTired is an application for cognitive fatigue assessment.

> It includes a stand-alone Android-App for fatigue detection and an additional part for data management and further analysis.

Reviewer: ```suggestion
It includes a stand-alone Android app for fatigue detection and an additional part for data management and further analysis.
```

> The project is structured in a client side (stand-alone App) implemented in Dart/Flutter and a server side (data management) in Golang.

Reviewer: ```suggestion
The project is structured as client side (stand-alone app) implemented in Dart/Flutter and a server side (data management) written in Golang.
```

> Dart/Flutter provide the possibility to simply add an iOS and / or Web-Application.

Reviewer: ```suggestion
Dart/Flutter provides the ability to simply add an iOS, desktop and / or web application besides Android.
```

> soTired is a remake of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.

Reviewer: ```suggestion
soTired is a rewrite of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.
```
